CIP-Improved_Rewards_Scheme_Parameters/README.md

```diff
@@ -0,0 +1,237 @@
+---
+CIP: 82
+Title: Improved Rewards Scheme Parameters
+Status: Project Area Not Enlisted for CIP Process
+Category: Reward-Sharing Scheme
+Authors:
+    - Tobias Fancee <tobiasfancee@gmail.com>
+Implementors:
+    - TBD
```

Comment:
```suggestion
Implementors: []
```
Unless I'm reading [CIP-0001 > Header Preamble](https://github.com/cardano-foundation/CIPs/tree/master/CIP-0001#header-preamble) wrong myself, this is the correct syntax for not having any implementors yet (cc @KtorZ - not sure since we don't have this syntax on any merged CIPs yet).